appveyor.yml

```diff
@@ -0,0 +1,65 @@
+branches:
+  only:
+    - master
+
+skip_non_tags: true
+
+version: "{branch}-{build}"
+
+image: Visual Studio 2015
+
+environment:
+  APPVEYOR_SAVE_CACHE_ON_ERROR: "true"
+  GTK_SDK_VERBOSE: 1
+  BUILD_DIR: C:\msys64\home\appveyor
+  BUILD_ROOT: exaile/tools/installer/_build_root
+  SDK_NAME: exaile-binenv-win
+  SDK_VER: 1
+  SDK_URL: https://github.com/exaile/exaile-binenv-win-appveyor/releases/download/$(SDK_NAME)-$(SDK_VER)/$(SDK_NAME)-$(SDK_VER).tar.xz
+  dist_name: $(APPVEYOR_PROJECT_NAME)-$(APPVEYOR_REPO_TAG_NAME)
+
+clone_folder: $(BUILD_DIR)\$(APPVEYOR_PROJECT_NAME)
+
+clone_depth: 1
+
+cache:
+  - C:\msys64\var\cache\pacman\pkg
+  - '%BUILD_DIR%\%SDK_NAME%-%SDK_VER%.tar.xz'
+
+init:
+  - set PATH=C:\msys64\usr\bin;%PATH%
+
+install:
+  - cmd: |
+      cd %BUILD_DIR%
+      git clone --depth=1 https://github.com/exaile/python-gtk3-gst-sdk.git
+      bash -lc "[[ -f $SDK_NAME-$SDK_VER.tar.xz ]] || time wget -O $SDK_NAME-$SDK_VER.tar.xz $SDK_URL"
+      bash -lc "time pacman --noconfirm --noprogressbar -Syuu"
+
+before_build:
+  - cmd: |
+      bash -lc "mkdir $BUILD_ROOT"
+      bash -lc "time tar -xf $SDK_NAME-$SDK_VER.tar.xz -C $BUILD_ROOT"
+
+build_script:
+  - cmd: |
+      bash -lc "cd exaile/tools/installer && time ../../../python-gtk3-gst-sdk/win_installer/build_win32_installer.sh"
```

Comment:
I wonder if setting `MAKEFLAGS=-j` can speed this up a bit.

Reply:
I dunno, you can try. It seems the VM has 2 CPUs, why not.

Reply:
[Before](https://ci.appveyor.com/project/X0rg/exaile/builds/22960607#L1347):
```
real	4m7.466s
user	0m20.471s
sys	0m39.088s
```
[After](https://ci.appveyor.com/project/X0rg/exaile/builds/22964889#L1658):
```
real	4m19.881s
user	0m21.022s
sys	0m46.107s
```

Reply:
[Before](https://ci.appveyor.com/project/X0rg/exaile/builds/22960607#L1347):
```
real	4m7.466s
user	0m20.471s
sys	0m39.088s
```
[After](https://ci.appveyor.com/project/X0rg/exaile/builds/22965096#L1398):
```
real	4m27.253s
user	0m22.522s
sys	0m45.907s
```

With `set MAKEFLAGS=-j0`, [it was complaining with `the '-j' option requires a positive integer argument`](https://ci.appveyor.com/project/X0rg/exaile/builds/22964889#L608).
With `MAKEFLAGS=-j$(nproc)`, there is no `make[2]` in log. It uses only one thread, while [`nproc` is `2` on this VM](https://ci.appveyor.com/project/X0rg/exaile/builds/22965010#L6).

Reply:
Well, added `MAKEFLAGS=-j2` in 6785443.